src/app/components/list/list.js

```diff
@@ -95,12 +95,11 @@ listControllers.controller('ListCtrl', ['$scope', '$routeParams', '$location', '
   $scope.isOwner = false;
   $scope.isFavorite = false;
 
-  $scope.$on('user-service-ready', function() { 
+  $scope.$on('user-service-ready', function() {
     $scope.list = List.get({'listId': $routeParams.list});
     var listCallback = function () {
       var req = [];
       req[$scope.list.type + 's.list'] = $scope.list._id;
-      userService.notify(req);
```

Comment:
If this is removed, when the list is pulled from the API, how do you notify the user controller that it needs to update its list of users ?